script.js

```diff
@@ -0,0 +1,52 @@
+let modalShow = document.querySelector("#modal-show");
+let modalClose = document.querySelector("#modal-close");
+let modalSuccess = document.querySelector("#modal-success");
+let modal = document.querySelector("#modal")
+
+modalShow.addEventListener("click", function(){
+    modal.classList.toggle("active")
+})
+
+modalSuccess.addEventListener("click", function(){
+    modal.classList.toggle("active");
+})
+
+modalClose.addEventListener("click", function () {
+  modal.classList.toggle("active");
+});
```

Comment:
as a part of best practice you can use arrow function on eventlisteners

instead of

```
modalShow.addEventListener("click", function(){
    modal.classList.toggle("active")
})
```

try

```
modalShow.addEventListener("click", () => {
    modal.classList.toggle("active")
})

```

this won't create any difference, but helps us to stick with convention :)